pkg/controller/jobs/pod/pod_controller_test.go

```diff
@@ -93,6 +93,53 @@ func TestPodsReady(t *testing.T) {
 	}
 }
 
+func TestPodIndex(t *testing.T) {
```

Comment:
Basically, we should add unit tests against top-level functions. In this case, we should add these tests to the `TestReconciler`.

If we continue to add unit testings against all unexported functions, maintenance costs for unit testings will significantly grow.